NullPointerException: Unable to start TextEditorActivity

###### Reproduction Steps
This crash is triggered with this shell instruction:

adb shell am start -n com.amaze.filemanager/com.amaze.filemanager.activities.TextEditorActivity

###### Stack Trace

E/AndroidRuntime( 2482): FATAL EXCEPTION: main
E/AndroidRuntime( 2482): Process: com.amaze.filemanager, PID: 2482
E/AndroidRuntime( 2482): java.lang.RuntimeException: Unable to start activity ComponentInfo{com.amaze.filemanager/com.amaze.filemanager.activities.TextEditorActivity}: java.lang.NullPointerException
E/AndroidRuntime( 2482): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2184)
E/AndroidRuntime( 2482): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2233)
E/AndroidRuntime( 2482): 	at android.app.ActivityThread.access$800(ActivityThread.java:135)
E/AndroidRuntime( 2482): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1196)
E/AndroidRuntime( 2482): 	at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime( 2482): 	at android.os.Looper.loop(Looper.java:136)
E/AndroidRuntime( 2482): 	at android.app.ActivityThread.main(ActivityThread.java:5001)
E/AndroidRuntime( 2482): 	at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime( 2482): 	at java.lang.reflect.Method.invoke(Method.java:515)
E/AndroidRuntime( 2482): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:785)
E/AndroidRuntime( 2482): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:601)
E/AndroidRuntime( 2482): 	at dalvik.system.NativeStart.main(Native Method)
E/AndroidRuntime( 2482): Caused by: java.lang.NullPointerException
E/AndroidRuntime( 2482): 	at com.amaze.filemanager.activities.TextEditorActivity.onCreate(TextEditorActivity.java:199)
E/AndroidRuntime( 2482): 	at android.app.Activity.performCreate(Activity.java:5231)
E/AndroidRuntime( 2482): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
E/AndroidRuntime( 2482): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2148)
E/AndroidRuntime( 2482): 	... 11 more


###### Debug info
App Version = 3.3.2

Android Version = 7.1.0

--- a/app/src/main/java/com/amaze/filemanager/activities/TextEditorActivity.java
+++ b/app/src/main/java/com/amaze/filemanager/activities/TextEditorActivity.java
@@ -193,6 +193,7 @@ public class TextEditorActivity extends ThemedActivity implements TextWatcher, V
         } else {
             Toast.makeText(this, R.string.no_file_error, Toast.LENGTH_LONG).show();
             finish();
+            return;
         }
 
         getSupportActionBar().setTitle(mFile.name);

